File: MyMindNotes/app/src/main/java/com/android/mymindnotes/model/retrofit/LoginApi.java
package com.android.mymindnotes.model.retrofit;

import com.android.mymindnotes.model.UserInfoLogin;

import java.util.Map;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface LoginApi {
    @POST("/api/member/login")
    Call<Map<String, Object>> login(@Body UserInfoLogin userinfologin);
}

File: MyMindNotes/app/src/main/java/com/android/mymindnotes/data/retrofit/LoginApi.kt
package com.android.mymindnotes.data.retrofit

import com.android.mymindnotes.data.retrofit.model.UserInfoLogin
import retrofit2.http.Body
import retrofit2.http.POST

interface LoginApi {
    @POST("/api/member/login")
    suspend fun login(
        @Body userInfoLogin: UserInfoLogin
    ): Map<String, Object>
}